docs/contributing/governance/veYFI-comp-summary.md

```diff
@@ -0,0 +1,293 @@
+    - **k**: Constant (approx. 4.7).
+    - **s**: Configurable scaling factor (default is 10).
+    - **x**: Ratio of veYFI supply to total YFI supply $$\frac{\text{veYFI Supply}}{\text{YFI Supply}}$$.
+
+  ![image](/img/charts/dyfi-redemption-chart.png)
+
+- **Implications**:
+  - **High veYFI Supply**: Lower discount, as more YFI is locked, indicating strong commitment.
+  - **Low veYFI Supply**: Higher discount, incentivizing users to lock more YFI.
+
+---
+
+## Epochs and Voting
+
+Governance and reward distribution operate on a structured timeline.
+
+### Epoch Timeline
+
+- **Duration**: Each epoch lasts **14 days**.
+- **Phases**:
+  - **Week 1**: Proposal submission period.
+  - **Week 2**: Voting period for gauge emissions and governance proposals.
+- **Voting Decay**: Voting power decays linearly during the last 24 hours of the voting period to prevent manipulation.
+
+### Gauge Emission Voting
+
+- **Allocation of dYFI**:
+  - **Total Emission**: Determined by the emission formula.
+  - **Reserved Allocations**:
+    - **5%** to YFI/ETH liquidity gauge.
+    - **5%** to dYFI/ETH liquidity gauge.
+  - **Remaining 90%**: Allocated based on veYFI holders' votes.
+- **Blank Votes**:
+  - **Option to Conserve Emission**: Voters can cast "blank" votes to reduce current epoch's emission.
+  - **Outcome of Blank Votes**:
+    - **Burned dYFI**: Permanently removes dYFI from circulation, extending the program's runway.
+    - **Deferred Emission**: Carries over dYFI to the next epoch's allocation.
+  - **Configurable Split**: The proportion of burned vs. deferred dYFI is adjustable through governance.
+
+### Governance Proposals
+
+- **Submission**:
+  - Any address holding at least **1 veYFI** can submit proposals during the first week of an epoch.
+  - Proposals are posted in the dedicated governance forum section.
+- **Voting and Approval**:
+  - Simple majority (>50% of votes) required for approval.
+  - **Quorum Requirement**: A minimum amount of veYFI must participate in the vote for it to be valid. The default quorum is **10 veYFI** but is configurable.
+- **Types of Proposals**:
+  - Adding or removing gauges.
+  - Adjusting system parameters.
+  - Implementing significant protocol changes.
+
+---
+
+## System Parameters and Governance Adjustments
+
+Certain aspects of the veYFI and dYFI system can be modified through governance to adapt to evolving needs.
+
+### Adjustable Parameters
+
+- **Quorum**: Minimum veYFI required for proposal approval (default: 10 veYFI).
+- **Blank Vote Burn Percentage**: Proportion of dYFI from blank votes that is burned (default: 50%).
+- **Scaling Factors**:
+  - **s (Discount Scaling)**: Affects the dYFI discount curve (default: 10).
+  - **c (Emission Scaling)**: Adjusts the emission rate of dYFI (default: 12).
+- **Liquidity Gauges**:
+  - Addresses of the YFI/ETH and dYFI/ETH liquidity gauges receiving reserved emissions.
+
+### Future Adjustments and Immutability
```

Comment:
Need input on whether this section is still current. References to epochs implies that these have probably elapsed as 12 epochs = 24 weeks or 6 months and we are currently on epoch 26 as far as I can tell.